### DISCUSSION TITLE
Welcome to ifunny Discussions!

### DISCUSSION THREAD

**Discussion by theabbie** [theabbie/ifunny#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Replikagami** [comment]
Ahh, I'd love to discuss a few things with you soon.

**Reply by theabbie** [reply]
sure

**Reply by Replikagami** [reply]
Unrelated to the discussion, I just typed it all out(An essay worth TwT) and didn't save it, so I have to redo everything... *Cry*   
Moving on. I noticed your history of contributions and hoped you might have such answers.
Going to shorten everything I typed earlier to just a specific topic I've wanted to understand.
I have been looking through some old documents for an iFunny bot and noticed something relating to an "Invite Box". I was curious just how much exists with this. I already understand that a certain ifunny bot( https://iFunny.co/user/MerpBot ) is able to read the API and see if someone is actively viewing a particular chat ( The .lurkers cmd). I was wondering if it's possible to read this "Invite Box" as well. Such as if there is a statistic that gives you a number for accepted invites/standing invites/declined invites. As well as if you're able to read the exact username/ID of each particular user holding such an invite. I'm also curious as to if it's possible to force an invite to stop working/force decline a user's invite to a chat.
\
Summary of question: Is it possible to interact with the API and see if anyone has an invite to your chat at that particular time.
